bundle/edu.gemini.pot/src/main/java/edu/gemini/spModel/gemini/niri/Niri.java

```diff
@@ -797,20 +793,6 @@ public ROIDescription(ROIDescription roid) {
         }
 
         /**
-         * Return the x start pixel.
-         */
-        public int getXStart() {
-            return _xStart;
-        }
-
-        /**
-         * Return the y start pixel.
-         */
-        public int getYStart() {
-            return _yStart;
-        }
-
-        /**
```

Comment:
The only exception I can recall is for the [TPE features](https://github.com/gemini-hlsw/ocs/blob/develop/bundle/jsky.app.ot/src/main/java/jsky/app/ot/OT.java#L223-L237), which are added via reflection.  If there are others it's probably better to find out now and deal with it anyway.